test/deploy/kubernetes/template-config/cluster_default.yaml

```diff
@@ -11,9 +11,10 @@ cluster:
       - name: hare
       - name: s3
         services:
-        - io
-        - auth
-        - bg_consumer
+        - s3server
```

Comment:
Why are changing in old infra? Will it work for 2 pod setup (Control and Storage PODs)

Reply:
yes , it is working for 2 pod set up also, we have tested it.